index.js

```diff
@@ -89,59 +86,51 @@ PersonaClient.prototype.validateToken = function (token, scope, overridingScope,
         return;
     }
 
-    var _this = this,
-        cacheKey = token + ((scope) ? "@" + scope : "");
-    // if we were given a scope then append the scope to the token to create a cachekey
-
-    this.debug("Validating token: " + cacheKey);
-    this.redisClient.get("access_token:" + cacheKey, function (err, reply) {
-        if (reply === "OK") {
-            _this.debug("Token " + cacheKey + " verified by cache");
-            next(null, VALIDATED_TYPES.CACHE);
-        } else {
-            var requestPath = _this.config.persona_oauth_route + token;
-            var usingScope = scope || overridingScope || null;
+    this.debug("Validating token: " + token);
```

Comment:
this will be bad for the logs as it is most likely debug will be left on within production